src/main/java/Duke.java

```diff
@@ -1,10 +1,128 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
+    private static void logoWithHello() {
+        String logo = " ____        _\n"
```

Comment:
Perhaps you should name the logo in capital letter as it is a constant? Also, would you like to consider naming the method with an action instead?

```suggestion
        String LOGO = " ____        _\n"
```